lib/rgeo/active_record/version.rb

```diff
@@ -2,6 +2,6 @@
 
 module RGeo
   module ActiveRecord
-    VERSION = "7.0.1"
+    VERSION = "8.0.0"
```

Comment:
@BuonOmo will we have to synchronize this with the postgis adapter PR that you have up? Do we need a major version bump here?